config/initializers/event_types.rb

```diff
@@ -0,0 +1,5 @@
+# load all event types so we can use it in e.g. drop-downs.
+
+Rails.application.reloader.to_prepare do
+  Dir["#{Rails.root}/app/events/*_event.rb"].each { |f| require f }
```

Comment:
Can/should this be done with `config.eager_load_paths`?

Reply:
Tried it and it doesn't seem to work, at least on development.